src/llama_server_context.cc

```diff
@@ -707,9 +708,9 @@ bool LlamaServerContext::UpdateSlots() {
           slot.state = SlotState::kProcessing;
           slot.command = SlotCommand::kNone;
           slot.Release();
-          // SendError(slot,
-          //            "Input prompt is too big compared to KV size. Please "
-          //            "try increasing KV size.");
```

Comment:
Why were these lines commented out? 🤔

Reply:
We did not support `SendError` for slot before, so somehow I commented out those lines :(